src/backend/joanie/core/management/commands/create_certificates.py

```diff
@@ -0,0 +1,223 @@
+            "--course",
+            help=(
+                "Accept a single or a list of course code to restrict review to "
+                "this/those course(s)."
+            ),
+        )
+        parser.add_argument(
+            "-p",
+            "--products",
+            "--product",
+            help=(
+                "Accept a single or a list of product uuid to restrict review to "
+                "this/those product(s)."
+            ),
+        )
+        parser.add_argument(
+            "-o",
+            "--orders",
+            "--order",
+            help=(
+                "Accept a single or a list of order uuid to restrict review to "
+                "this/those order(s)."
+            ),
+        )
+
+    # pylint: disable=too-many-locals
+    def handle(self, *args, **options):
+        """
+        Retrieve all certifying products then for each of them check eligibility for
+        certification of all related orders.
+        If `order` option is used, this order is directly retrieve.
+        """
+        order_uids = None
+        product_uids = None
+        course_codes = None
+
+        if options["orders"]:
+            order_uids = (
+                options["orders"]
+                if isinstance(options["orders"], list)
+                else [options["orders"]]
+            )
+
+        total = 0
+        if order_uids:
+            filters = {
+                "is_canceled": False,
+                "certificate__isnull": True,
+                "product__type__in": enums.PRODUCT_TYPE_CERTIFICATE_ALLOWED,
+                "uid__in": order_uids,
+            }
+
+            try:
+                orders = [
+                    order
+                    for order in models.Order.objects.select_related(
+                        "course__organization"
+                    ).filter(**filters)
+                    if order.state == enums.ORDER_STATE_VALIDATED
+                ]
+            except models.Order.DoesNotExist:
+                return str(total)
+
+            for order in orders:
+                result = self._generate_certificate_for_order(order)
+                total += result
+
+            return str(total)
+
+        if options["courses"]:
+            course_codes = (
+                options["courses"]
+                if isinstance(options["courses"], list)
+                else [options["courses"]]
+            )
+
+        if options["products"]:
+            product_uids = (
+                options["products"]
+                if isinstance(options["products"], list)
+                else [options["products"]]
+            )
+
+        filters = {"type__in": enums.PRODUCT_TYPE_CERTIFICATE_ALLOWED}
+        if course_codes:
+            filters.update({"courses__code__in": course_codes})
+        if product_uids:
+            filters.update({"uid__in": product_uids})
+
+        products = models.Product.objects.filter(**filters)
+
+        for product in products:
```

Comment:
be gentle on the RAM as the number of order may be big!?
```suggestion
        for product in products.iterator():
```